docs/auth/federated-auth.md

```diff
@@ -224,6 +224,32 @@ Future<UserCredential> signInWithApple() async {
 }
 ```
 
+#### Apple platform sign-in only
+
+Apple sign-in on iOS+ platforms can also be achieved with the following method:
+
+```dart
+// Create a nonce
+String rawNonce = createNonce();
+// Create a hash SHA-256 string of the nonce. Consider using the `crypto` package on pub.dev registry
+String hashSHA256String = createHashSHA256String(rawNonce);
+// Use the hash SHA-256 string of the nonce to get the idToken. Consider using the `sign_in_with_apple` plugin on pub.dev registry
```

Comment:
Not sure how I feel about suggesting sign_in_with_apple since we are supporting signin without it, we have the rawNonce generated on the native side, not sure why we need to generate it on Dart side for this 🤔

Reply:
Yeah, I agree, but not sure what to do about it really as they're needed as inputs 🤔. My sense is that users are going to use the much easier way to apple sign in (`AppleProvider`) rather than this method. 

Happy to remove but I think it would confuse users on what to do if don't at least document the flow.

Reply:
Yes I guess we can put it that way and see how it goes with issues.